index.ts

```diff
@@ -182,6 +182,16 @@ for (const [key, data] of yamlEntries('features')) {
     features[key] = data;
 }
 
+// Assert that discouraged feature's alternatives are valid
+for (const [id, feature] of Object.entries(features)) {
+    for (const alternative of feature.discouraged?.alternatives ?? []) {
+        console.log(`Confirming ${alternative} in feature set`);
+        if (!(alternative in features)) {
+            throw new Error(`${id}'s alternative "${alternative}" is not a valid feature ID`);
+        }
+    }
+}
+
```

Comment:
This is succeeding the test workflow on GitHub Actions, but failing locally (it **should** fail). I think this is somewhat worrying. Maybe someone else can test this locally to reproduce?

Reply:
@ddbeck If I'm not mistaken, `actions/checkout` checks out a commit that is the result of merging the PR branch into the target branch, and not on the PR branch itself. Since `destructuring` is now present on main, it doesn't fail. To test, you could merge main into your local branch and verify it passes, and then push a commit with a fake alternative id and verify it fails successfully.

Reply:
Ah, this makes sense now. Thanks for the explanation, @jamesnw!